test/test_device_hydro_generation_constructors.jl

```diff
@@ -819,152 +832,3 @@ end
-    model = DeviceModel(HydroEnergyReservoir, HydroCommitmentRunOfRiver)
-
-    # Parameters Testing
-    op_problem =
-        OperationsProblem(MockOperationProblem, DCPPowerModel, c_sys5_hyd; use_parameters = true)
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, true, 96, 0, 72, 0, 24, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Parameters Testing
-    op_problem = OperationsProblem(MockOperationProblem, DCPPowerModel, c_sys5_hyd)
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, false, 96, 0, 48, 0, 24, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Forecast Testing
-    op_problem = OperationsProblem(
-        MockOperationProblem,
-        DCPPowerModel,
-        c_sys5_hyd;
-        use_parameters = true,
-        use_forecast_data = false,
-    )
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, true, 4, 0, 3, 0, 1, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Forecast - No Parameters Testing
-    op_problem = OperationsProblem(
-        MockOperationProblem,
-        DCPPowerModel,
-        c_sys5_hyd;
-        use_forecast_data = false,
-    )
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, false, 4, 0, 2, 1, 1, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-end
-
-@testset "Hydro DCPLossLess HydroEnergyReservoir with HydroCommitmentReservoirlFlow Formulations" begin
-    model = DeviceModel(HydroEnergyReservoir, HydroCommitmentReservoirFlow)
-
-    # Parameters Testing
-    op_problem =
-        OperationsProblem(MockOperationProblem, DCPPowerModel, c_sys5_hyd; use_parameters = true)
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, true, 96, 0, 72, 0, 24, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Parameters Testing
-    op_problem = OperationsProblem(MockOperationProblem, DCPPowerModel, c_sys5_hyd)
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, false, 96, 0, 48, 0, 24, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Forecast Testing
-    op_problem = OperationsProblem(
-        MockOperationProblem,
-        DCPPowerModel,
-        c_sys5_hyd;
-        use_parameters = true,
-        use_forecast_data = false,
-    )
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, true, 4, 0, 3, 0, 1, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-    # No Forecast - No Parameters Testing
-    op_problem = OperationsProblem(
-        MockOperationProblem,
-        DCPPowerModel,
-        c_sys5_hyd;
-        use_forecast_data = false,
-    )
-    mock_construct_device!(op_problem, model)
-    moi_tests(op_problem, false, 4, 0, 2, 1, 1, true)
-    psi_checkobjfun_test(op_problem, GAEVF)
-
-end
-
-@testset "Hydro DCPLossLess HydroEnergyReservoir with HydroDispatchReservoirStorage Formulations" begin
```

Comment:
This is a valid test that needs to be enabled

Reply:
present on line 553